File: 백준/Bronze/13136. Do Not Touch Anything/Do Not Touch Anything.cs
string[] inputs = Console.ReadLine().Split();

double R = double.Parse(inputs[0]);
double C = double.Parse(inputs[1]);
double N = double.Parse(inputs[2]);

long RCCTV = (long)Math.Ceiling(R / N);
long CCCTV = (long)Math.Ceiling(C / N);

long CCTV = RCCTV * CCCTV;

Console.WriteLine(CCTV);

File: 백준/Bronze/13136. Do Not Touch Anything/Do Not Touch Anything.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.Scanner;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) {
        Scanner sc =new Scanner(System.in);
        double R = sc.nextDouble();
        double C = sc.nextDouble();
        double N = sc.nextDouble();
        
        long RCCTV = (long) Math.ceil(R / N);
        long CCCTV = (long) Math.ceil(C / N);

        //넓이
        long CCTV = RCCTV * CCCTV;

        System.out.println(CCTV);
    }
}